src/dropbox.ml

```diff
@@ -292,8 +292,21 @@ module type S = sig
                       -> t -> shared_folder list Lwt.t
 
   module Fileops : sig
+    type root_fileops = [ `Auto | `Dropbox | `Sandbox ]
```

Comment:
Well, to not be mistaken with root from metadata type. But in module Fileops, it's indeed useless to add _fileops.